repotracker/repotracker.go

```diff
@@ -289,16 +289,16 @@ func (repoTracker *RepoTracker) StoreRevisions(ctx context.Context, revisions []
 					"project_identifier": ref.Identifier,
 					"revision":           revision,
 				}))
-				if pInfo.Project == nil {
-					grip.Error(message.WrapError(err, message.Fields{
-						"message":            fmt.Sprintf("unable to find project config for revision %s", revision),
-						"runner":             RunnerName,
-						"project":            ref.Id,
-						"project_identifier": ref.Identifier,
-					}))
-				}
 				return err
 			}
+		} else if pInfo.Project == nil {
+			grip.Error(message.WrapError(err, message.Fields{
```

Comment:
We can't wrap error here, because we've guaranteed in the if/else that there _is_ no error, so this will noop. Should just use message.Fields.